### DISCUSSION TITLE
Uninstall "built in" extensions

### DISCUSSION THREAD

**Discussion by onx2** [zed-industries/zed#16217] [Feature Requests]
Is it possible to "uninstall" the built-in extensions like eslint/prettier? I will never use them in my project but don't see an option to remove them from my Zed installation. 

I think adding the ability to choose the "built-ins" to some extent would align well with Zed's philosophy of speed -- If I don't need them its effectively bloatware.

What are everyone's thoughts on these capabilities? I used eslint/prettier as an example but this could be for anything that isn't related to core requirements for Zed to run.

**Comment by ghost** [comment]
I too find it rather odd to include a specific software package by default like this, and it's rather annoying that the only alternative seems to be to explicitly state all language servers and prepend `"!eslint"` AND add `"prettier": { "allow": false }` 😮

**Comment by JosephTLyons** [comment]
Thanks for reaching out. We want to better support our community, and the best way for us to do that is through our official Discord. The team is active there and happy to help.

We’re closing the “Help and General Q&A” category on GitHub Discussions and directing all support questions to Discord going forward.

https://zed.dev/community-links
